src/debug/ee/controller.cpp

```diff
@@ -4957,6 +4958,7 @@ DebuggerBreakpoint::DebuggerBreakpoint(Module *module,
 
     BOOL bindAcrossAllJittedInstances = !native;
```

Comment:
this should be bindAcrossAllJittedInstances = nativeCodeBindAllVersions 

... or just change all references to bindAcrossAllJittedInstances -> nativeCodeBindAllVersions. There is no need to have two identical variables.